src/frontends/pytorch/src/node_context.cpp

```diff
@@ -147,6 +148,18 @@ std::shared_ptr<ov::Model> NodeContext::convert_subgraph(size_t index) const {
 OutputVector NodeContext::inputs() const {
     OutputVector res;
     for (size_t i = 0; i < m_decoder_inputs.size(); i++) {
+        if (input_is_none(i)) {
```

Comment:
This will work for every case, but in TS case the None node will be already inserted in tensor_map. Lets do this only when we didn't find the node in tensor_map

Reply:
made changes more safe